core/templates/dev/head/components/ck-editor-helpers/ck-editor-5-rte.directive.ts

```diff
@@ -75,27 +80,26 @@ oppia.directive('ckEditor5Rte', [
         // a div with a CSS class.
         var blockOverlayRule = ' div(oppia-rte-component-overlay);';
         // Put all the rules together.
-        var extraAllowedContentRules = (
-          componentRule + inlineWrapperRule + blockWrapperRule + blockOverlayRule);
+        var extraAllowedContentRules = ( componentRule +
```

Comment:
Should break after '('. Not sure why this got changed.

Reply:
The line exceeds 80 chars so I split it that way is it right?

Reply:
If the contents of a paren spread out over more than one line, always break after the opening paren. You can add a further line break after one of the "+" signs if needed.